### DISCUSSION TITLE
Support hashed passwords

### DISCUSSION THREAD

**Discussion by cdalexndr** [grafana/grafana#38344] [Dashboards, Panels & Visualizations]
Configuring passwords (security.admin_password & others) should support hashed passwords such as outputed from `openssl passwd`, instead of clear text.

**Comment by TuningYourCode** [comment]
I'm not sure how secure the generated hashes by `openssl passwd` are but i would prefer some more "common" (or multiply options for hashed passwords) formats.
e.g. our user already generate hashes of their passwords by `mkpasswd -m sha-512` (if command not found install whois package on debian) which we can use for the linux user account in our config management tool (puppet) to create user without knowing the password of the user.

Our use case would be too, that we want to create grafana users without knowing their passwords. Would be super cool if we wouldn't have to collect multiply formats of password hashes :)

**Comment by hvhaugwitz** [comment]
What is the state of this feature request?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82981. Feel free to continue the discussion around this there. Thank you!
